doc/ensemble.rst

```diff
@@ -0,0 +1,739 @@
+********
+Ensemble
+********
+
+========
+Overview
+========
+An ``Ensemble`` is a SmartSim object comprised of multiple
+:ref:`Model<model_api>` objects, each representing an individual
+simulation. An Ensemble can be be reference as a single instance and
+enables users to launch a group of simulations
+at runtime. Ensembles can be launched with other SmartSim
+entities and ML infrastructure to build AI-enabled workflows.
+The :ref:`Ensemble API<ensemble_api>` offers key Ensemble features,
+including helper functions to:
+
+- :ref:`Ensemble.add_ml_model()<load_ml_model>`: load a TF, TF-lite, PT, or ONNX model into the database at runtime.
+- :ref:`Ensemble.add_function()<load_tf_function>`: load TorchScript functions to launch with each entity belonging to the ensemble.
+- :ref:`Ensemble.add_script()<load_script>`: load TorchScripts to launch with each entity at runtime.
+- :ref:`Ensemble.enable_key_prefixing()<prefix_ensemble>`: prevent key overlapping and enable ensemble members to run the same code.
+- :ref:`Ensemble.attach_generator_files()<attach_file>`: attach configuration files to assign ensemble member parameters or load parameters to file.
+
+An Ensemble is created through the ``Experiment.create_ensemble()`` function.
+There are **three** Ensemble configuration that users can adjust to customize
```

Comment:
- i think configuration should be plural

- Could we restate more clearly?

  - "Ensembles can be customized to exhibit three high-level behaviors"
  - "Ensembles can be configured to behave in the following ways:" 

As-is, it sounds like I need to modify one of three configuration values to achieve a behavior.